ERCS/erc-7540.md

```diff
@@ -272,6 +272,63 @@ MUST NOT revert unless due to integer overflow caused by an unreasonably large i
       type: uint256
 ```
 
+#### `isOperator`
+
+Returns `true` if the `operator` is approved as an operator for an `owner`.
+
+```yaml
+- name: isOperator
+  type: function
+  stateMutability: view
+
+  inputs:
+    - name: owner
+      type: address
+    - name: operator
+      type: address
+
+  outputs:
+    - name: status
+      type: bool
+```
+
+#### `setOperator`
+
+Grants or revokes permissions for `operator` to manage Requests on behalf of the `msg.sender`.
```

Comment:
should this be: "Grants or revokes permissions for `operator` to manage Requests on behalf of the `owner`"